public/app/features/dashboard-scene/utils/variables.ts

```diff
@@ -138,7 +138,7 @@ export function createSceneVariableFromVariableModel(variable: TypedVariableMode
       filters: variable.filters ?? [],
       baseFilters: variable.baseFilters ?? [],
       defaultKeys: variable.defaultKeys,
-      allowCustomValue: variable.allowCustomValue ?? true,
+      allowCustomValue: variable.allowCustomValue,
```

Comment:
I assume MultiValueVariable treats undefined as true for this option?

Reply:
Yes, all supporting variables default to true, e.g. [here](https://github.com/grafana/scenes/pull/956/files#diff-4aa775b002cad3f443d0c382bcd93fb17e7902b8c87ea1b6276725503905b8f4R104)